EduVPN/Models/ServerResponse.swift

```diff
@@ -180,3 +184,82 @@ private enum SecondLevelKeys: String, CodingKey {
     case data
     case error
 }
+
+// Responses for APIv3
+
+protocol ServerResponseAPIv3: ServerResponse {
+}
+
+// Parse response to a /info request
+//
+// Example response:
+// {
+//     "info": {
+//         "profile_list": [
+//             {
+//                 "display_name": {
+//                     "en": "Employees",
+//                     "nl": "Medewerkers"
+//                 },
+//                 "profile_id": "employees"
+//             },
+//             {
+//                 "display_name": "Administrators",
+//                 "profile_id": "admins"
+//             }
+//         ]
+//     }
+// }
+
+struct InfoResponse: ServerResponseAPIv3, Decodable {
+    let data: [Profile]
+
+    enum TopLevelKeys: String, CodingKey {
+        case info
+    }
+
+    enum SecondLevelKeys: String, CodingKey {
+        case profile_list // swiftlint:disable:this identifier_name
```

Comment:
You might want to use `JSONDecoder.KeyDecodingStrategy.convertFromSnakeCase` instead.

Reply:
I think it's better the names are explicitly specified in code.